prac_05/kivy_layout.kv

```diff
@@ -0,0 +1,21 @@
+# This is a Kv file.
+# It contains the declarations of the widgets that will be displayed.
+BoxLayout:
+    orientation: 'vertical'
+    Label:
+        text: app.status_text
+        font_size: 60
+        size_hint_y: 0.2
+    BoxLayout:
+        orientation: 'horizontal'
+        Button:
+            text: "Up"
+            on_press: app.handle_press(1)
+        Button:
+            text: "Down"
+            on_press: app.handle_press(-1)
+    BoxLayout:
+        id: names_box
+    Label:
```

Comment:
How did you go with this one? I got stuck in a thought loop trying to work out where to put lines 19-21 or ages!

Reply:
Yeah I got confused too. Took me a minute to figure this one out :)